board/safety/safety_toyota.h

```diff
@@ -87,14 +98,21 @@ static void toyota_rx_hook(const CANPacket_t *to_push) {
     }
 
     // enter controls on rising edge of ACC, exit controls on ACC off
-    // exit controls on rising edge of gas press
-    if (addr == 0x1D2) {
-      // 5th bit is CRUISE_ACTIVE
+    if (!toyota_secoc_car && (addr == 0x1D2)) {
+      bool cruise_engaged = GET_BIT(to_push, 5U);
+      pcm_cruise_check(cruise_engaged);
+    }
+    if (toyota_secoc_car && (addr == 0x176)) {
```

Comment:
so many of these are different, I'd consider a big `if secoc; else`. not sure that it's cleaner though, up to you

Reply:
I did that in the car port, but in Panda safety, it's nice having all the alternate-gas, alternate-brake, etc code in a place where it can be seen and understood together.

I might feel differently if Panda had the ability (and test coverage) to reject entry into a safety mode if openpilot sent nonsense flag combinations.

Reply:
I also think combining these if `toyota_secoc_car` checks in one large if else is cleaner

Reply:
Gave it a try and I think it does look better, so I'm onboard.